Configurations.md

```diff
@@ -1766,7 +1766,7 @@ See also: [`match_block_trailing_comma`](#match_block_trailing_comma).
 What Write Mode to use when none is supplied: Replace, Overwrite, Display, Diff, Coverage
 
 - **Default value**: `"Overwrite"`
-- **Possible values**: `"Checkstyle"`, `"Coverage"`, `"Diff"`, `"Display"`, `"Overwrite"`, `"Plain"`, `"Replace"`
+- **Possible values**: `"Checkstyle"`, `"Coverage"`, `"Diff"`, `"Display"`, `"Overwrite"`, `"Plain"`, `"Replace"`, `"Modified"`
```

Comment:
Given this is not meant to be a user-facing option, it shouldn't be documented here.

Reply:
Done.